.github/workflows/lint.yml

```diff
@@ -160,9 +166,13 @@ jobs:
           pip install -r requirements-flake8.txt
           flake8 --version
       - name: Run flake8
+        env:
+          # See note [local lint]
+          LOCAL_FILES: ""
         run: |
           set -eux
-          flake8 | tee "${GITHUB_WORKSPACE}"/flake8-output.txt
+          # shellcheck disable=SC2086
+          flake8 $LOCAL_FILES  # | tee "${GITHUB_WORKSPACE}"/flake8-output.txt
```

Comment:
Please [fix this properly](https://github.com/koalaman/shellcheck/wiki/SC2086) rathe rthan ignoring SC2086.

Reply:
The word splitting is intended here, flake8/mypy need to see every file as a separate arg. This will break if we have any file names with spaces, but we don't currently (we could add a lint rule for it). I can't think of a good way to fix it since we're passing in the file names in a string via the env variable, so we have to split it somehow

Reply:
I understand that, but I was under the impression that [the wiki page](https://github.com/koalaman/shellcheck/wiki/SC2086#exceptions) suggests several solutions; if those are too burdensome, though, I suppose it's fine to ignore the warning here.

In the past I've solved this via Bash arrays as the wiki page suggests: https://github.com/pytorch/pytorch/pull/55165/files#diff-f99b45c851e816508bc55ea74a21725426c1c8c70abdee4a8d0410ec1ee1c873 However, I guess that approach may not work here, so you may be right.

Reply:
Yeah I think those don't work I think since environment variables are all just strings and can't use other Bash types (like arrays as in the wiki)